posthog/migrations/0360_add_plugin_config_ui_fields.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.19 on 2023-11-03 12:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0359_team_external_data_workspace_id"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="deleted",
+            field=models.BooleanField(blank=True, default=False, null=True),
```

Comment:
I don't think we should set `null` here. `default=False` should be enough for the migration I think? I'm imagining this roughly translates to: `ALTER TABLE posthog_pluginconfig ADD COLUMN deleted DEFAULT FALSE` which means PG will store `FALSE` for all rows at the time of the migration, without needing a rewrite.

Reply:
It could be that django complains though, but from Postgres' side this should be fine.

Reply:
exactly I was worried about the migration stuff, would be more than happy to change it to default=False if that's safe

Reply:
https://posthog.slack.com/archives/C045QJCUGQ4/p1699355486469589